Event mapping: Tiktok Ads doc (#779)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/tiktok-ads.mdx

- **Hash Contextual User Properties (SHA-256)**: If this setting is enabled, RudderStack will hash the contextual user properties like `external_id`, `email`, `phone_number` from the raw format to the SHA-256 format.

> - **Mapping to trigger the TikTok Ads standard events for the respective Events**: Enter the event name and pick a corresponding <Link to="https://ads.tiktok.com/help/article?aid=10028">TikTok Ads standard event</Link> from the dropdown to be triggered when that event is called.

Reviewer: ```suggestion
- **Mapping to trigger the TikTok Ads standard events for the respective Events**: Enter the event name and select the corresponding <Link to="https://ads.tiktok.com/help/article?aid=10028">TikTok Ads standard event</Link> from the dropdown to be triggered when that event is called.
```

<div class="successBlock">
You can specify multiple <strong>standard events</strong> for one <strong>event name</strong> and vice versa.
</div>

> 

Reviewer: ```suggestion

<div class="infoBlock">
This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
```

- **Hash Contextual User Properties (SHA-256)**: If this setting is enabled, RudderStack will hash the contextual user properties like `external_id`, `email`, `phone_number` from the raw format to the SHA-256 format.

- **Mapping to trigger the TikTok Ads standard events for the respective Events**: Enter the event name and select the corresponding <Link to="https://ads.tiktok.com/help/article?aid=10028">TikTok Ads standard event</Link> from the dropdown to be triggered when that event is called.

<div class="successBlock">
You can specify multiple <strong>standard events</strong> for one <strong>event name</strong> and vice versa.
</div>

<div class="infoBlock">

> This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.

Reviewer: ```suggestion
This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.

```

- **Hash Contextual User Properties (SHA-256)**: If this setting is enabled, RudderStack will hash the contextual user properties like `external_id`, `email`, `phone_number` from the raw format to the SHA-256 format.

- **Mapping to trigger the TikTok Ads standard events for the respective Events**: Enter the event name and select the corresponding <Link to="https://ads.tiktok.com/help/article?aid=10028">TikTok Ads standard event</Link> from the dropdown to be triggered when that event is called.

<div class="successBlock">
You can specify multiple <strong>standard events</strong> for one <strong>event name</strong> and vice versa.
</div>

<div class="infoBlock">

> This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.

Reviewer: ```suggestion
This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
```
```suggestion
This setting is applicable only when sending the events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
</div>
```
